### PR TITLE
New links to RateLimitPolicy guide

### PR DESCRIPTION
**Pull request by jasonmadigan**
Related: https://github.com/Kuadrant/multicluster-gateway-controller/pull/612

Should not merge until this lands

/onhold

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -93,9 +93,7 @@ nav:
       - 'Multicluster Gateways Walkthrough': multicluster-gateway-controller/docs/how-to/multicluster-gateways-walkthrough.md
       - 'Metrics Walkthrough': multicluster-gateway-controller/docs/how-to/metrics-walkthrough.md
     - 'Rate Limiting':
-      - kuadrant-operator/doc/user-guides/simple-rl-for-app-developers.md
-      - kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md
```
Comment (guicassolato) [inline mkdocs.yml:97]: No RLP user guide for platform engineers anymore?

File: mkdocs.yml [hunk 2]
```diff
@@ -93,9 +93,7 @@ nav:
...
-      - kuadrant-operator/doc/user-guides/authenticated-rl-for-app-developers.md
```
Comment (guicassolato) [inline mkdocs.yml:98]: I'm guessing this one will come back after AuthPolicy v2 maybe?

File: mkdocs.yml (hunk 1 above)
Comment (jasonmadigan) [reply mkdocs.yml:97]: good point, will keep kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md

**Comment by jasonmadigan** [comment]
merging to go with https://github.com/Kuadrant/multicluster-gateway-controller/pull/612

**Pull request merged**